src/assets/css/site.css

```
@@ -366,25 +395,100 @@ html.js .js-fallback-notice {
+  position: absolute;
+  inset: 0;
+  z-index: 0;
+  background: var(--color-reader);
+  border-radius: var(--radius-md);
+  pointer-events: none;
+}
+
+.app-shell__content-corner-mask {
+  --reader-corner-mask-size: var(--radius-md);
+  --reader-corner-mask-fill: var(--color-bg);
+  position: sticky;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)